src/Microsoft.DotNet.Build.Tasks/PackageFiles/packages.Unix.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="dnx-mono" version="1.0.0-rc2-16128" />
```

Comment:
I think as part of our workflow shift we should be able to remove all these *.config files that are used for restoring. We will no longer need to restore dnx only dotnet cli and the fact it uses dnx behind the scenes right now we don't need to care about.